Tactical/Keys.cpp

```diff
@@ -1190,9 +1190,6 @@ BOOLEAN ModifyDoorStatus( INT32 sGridNo, BOOLEAN fOpen, BOOLEAN fPerceivedOpen )
 
 	if ( pBaseStructure == NULL )
 	{
-#if 0

```

Comment:
Preserve these function calls and modify them to be at appropriate debug level instead of MSG_BETAVERSION?